lib/zendesk_apps_support/package.rb

```diff
@@ -84,8 +85,12 @@ def files
       files
     end
 
+    def text_files
+      @text_files ||= files.select { |f| f =~ %r{.*((ht|x)ml?|js(on)?)$} } # match .html, .xml, .js and .json
```

Comment:
This is very cute but I wonder if it'd be more readable as `(html|xml|js|json)$`?

Reply:
Fine.

Reply:
I mean my regex would've matched on `.xm` files as well for [Triton FastTracker 2 extended modules](https://en.wikipedia.org/wiki/XM_(file_format) containing exposed secrets, but we'll just ignore that feature then